subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidPluginsSmokeTest.groovy

```diff
@@ -28,7 +28,7 @@ import spock.lang.Unroll
  *
  */
 class AndroidPluginsSmokeTest extends AbstractSmokeTest {
-    public static final ANDROID_BUILD_TOOLS_VERSION = '26.0.2'
+    public static final ANDROID_BUILD_TOOLS_VERSION = '27.0.3'
```

Comment:
@bmuschko - should this tool version be updated?

Reply:
Yes, we'll want to be on the cutting edge to ensure that new versions of Android don't break for our users.